backend/src/store/room.go

```diff
@@ -3,29 +3,32 @@ package store
 import (
 	"encoding/json"
 	"sync"
+	"time"
 
 	log "github.com/sirupsen/logrus"
 	"github.com/sprinteins/poker/src/x/types"
 )
 
 type Room struct {
-	id         types.ID
-	players    []*Player
-	mtx        sync.Mutex
-	status     Status
-	cards      Cards
-	autoReveal bool
+	id              types.ID
+	players         []*Player
+	mtx             sync.Mutex
+	status          Status
+	cards           Cards
+	autoReveal      bool
+	autoRevealTimer int
```

Comment:
I think the name is a bit misleading. It is not the timer itself it is the timeout(?) 
so something like `autoRevealTimeout`. However it is separate from the timer so I think I would chose a different name. How about something like `timeboxTimeout` or just `timebox`?

I would like to see somewhere the unit too. I can imagine it like either in the name `timeboxInSeconds`, `timeboxSeconds` or as a type:
 ```go
 type second = int
 …
 timebox second
 ```